core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -0,0 +1,299 @@
+package forwarders
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	"github.com/ethereum/go-ethereum/accounts/abi/bind"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/pkg/errors"
+	evmclient "github.com/smartcontractkit/chainlink/core/chains/evm/client"
+	evmlogpoller "github.com/smartcontractkit/chainlink/core/chains/evm/logpoller"
+	evmtypes "github.com/smartcontractkit/chainlink/core/chains/evm/types"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/authorized_forwarder"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/authorized_receiver"
+	"github.com/smartcontractkit/chainlink/core/internal/gethwrappers/generated/offchain_aggregator_wrapper"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/chainlink/core/utils"
+	"github.com/smartcontractkit/sqlx"
+)
+
+// Config encompasses config used by fwdmgr
+//go:generate mockery --recursive --name Config --output ./mocks/ --case=underscore --structname Config --filename config.go
```

Comment:
Is this mock being used for anything interesting? If not, `pgtest.NewPGCfg(bool)` is already available.

Reply:
Nothing at the moment, but this config should include a number of options for forwarding in coming PRs

Reply:
Happy to  change for now but seemed like a good direction to go with mocking

Reply:
Whichever you think will be useful :shrug:
I tend to prefer simple struct implementations for things like configs that are static and don't need to record behevior, but not a big deal.

Reply:
yeah +1 on concrete structs they are faster to change and easier to reason about, I'd only introduce mocking if you absolutely need it

Reply:
Done